src/libraries/ref.proj

```diff
@@ -0,0 +1,36 @@
+<Project InitialTargets="IncludeProjectReferences">
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'"> 
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)\System.Utf8String.Experimental\ref\System.Utf8String.Experimental.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Project Condition="'$(DirectoryToBuild)' == ''" Include="$(MSBuildThisFileDirectory)*\ref\*.*proj" Exclude="@(ProjectExclusions)" />
+    <Project Condition="'$(DirectoryToBuild)' != ''" Include="$(DirectoryToBuild)\ref\*.*proj" />
+  </ItemGroup>
+
+  <Target Name="IncludeProjectReferences">
+    <ItemGroup>
+      <ProjectReference Include="@(Project)" />
+    </ItemGroup>
+  </Target>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+  
+  <Target Name="Build" DependsOnTargets="BuildAllProjects" />
+  <Target Name="BuildAllProjects">
+    <MSBuild Targets="Build"
+             Projects="@(Project)"
+             Properties="BuildAllProjects=true;%(Project.AdditionalProperties)"
+             BuildInParallel="true"
+             ContinueOnError="ErrorAndStop" />
+
+    <!-- Given we ErrorAndContinue we need to propagate the error if the overall task failed -->
```

Comment:
Looks from above that now we have it as ErrorAndStop, so we probably don't need this last Error task any more?

Reply:
I believe in the past I copied that block from dir.traversal.targets. Let's try it out.

Reply:
Oh I see, for testing it is ErrorAndContinue which makes sense.